include/ignition/math/Color.hh

```diff
@@ -126,11 +126,17 @@ namespace ignition
       public: Color &operator=(const Color &_pt);
 
       /// \brief Array index operator
-      /// \param[in] _index Color component index(0=red, 1=green, 2=blue)
+      /// \param[in] _index Color component index(0=red, 1=green, 2=blue, 3=alpha)
       /// \return r, g, b, or a when _index is 0, 1, 2 or 3. A NAN_F value is
       /// returned if the _index is invalid
       public: float operator[](const unsigned int _index);
 
+      /// \brief Array index operator, const version
+      /// \param[in] _index Color component index(0=red, 1=green, 2=blue, 3=alpha)
```

Comment:
codecheck in github actions is complaining about line lengths on two lines in this file